packages/server/src/arpa_reporter/db/arpa-subrecipients.js

```diff
@@ -14,6 +14,21 @@ function baseQuery(trns) {
         .leftJoin('users AS users2', 'arpa_subrecipients.updated_by', 'users2.id');
 }
 
+async function archiveRecipient(id, { updatedByUser }, trns = knex) {
+    const query = trns('arpa_subrecipients')
+        .where('id', id)
+        .returning('*');
+
+    if (updatedByUser) {
```

Comment:
Should this be required? At the very least, we might want to always update the `updated_at` field.

Reply:
I don't have a strong opinion here.  I just copied this pattern from the `updateRecipient` method in the same file below.  I figure they should at least be consistent with each other.

I did mark session as required in the API endpoint though, so it should always be set in practice.